src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -279,6 +279,8 @@ Agents
     CETypes = any
     # List of CEs that will be treated by this SiteDirector ("any" can refer to any type of CE defined in the CS)
     CEs = any
+    # List of Tags that are required to be present in the CE/Queue definition
+    Tags =
```

Comment:
Should it be more like "RequiredTags"?

Reply:
No, these are not the Tags required by the site to be present in the Jobs. This is the semantics of the RequiredTags option